build/Dockerfile.spire-registration

```diff
@@ -1,4 +1,4 @@
-FROM lobkovilya/spire-server:kind as builder
+FROM gcr.io/spiffe-io/spire-server:1c28d02f1ac980cf0fb42de78b1fea9c2f172e4d as builder
```

Comment:
If the 0.8.4 release has the needed fixes... shall we just go with that?

Reply:
Yes for now, but JWT tokens needs Spire PR https://github.com/spiffe/spire/pull/1169, which is not under release yet. 
And are we sure that 0.8.4 has PR that fixes crio stuff?